command/root.go

```diff
@@ -351,3 +352,37 @@ func determineEditor(cmd *cobra.Command) (string, error) {
 
 	return editorCommand, nil
 }
+
+func ExpandAlias(args []string) ([]string, error) {
+	// TODO i don't love that context is being init'd twice. My reading of the code is that it is
+	// idempotent and okay but I don't know if that's ok to depend on.
```

Comment:
I think this is fine for now! In the future if we move to factory-based command initialization, we will pass a single instance of Context or Config through the factory anyway, so the double-initialization will be eliminated.